pom.xml

```diff
@@ -17,41 +21,44 @@
             <url>http://www.opensource.org/licenses/mit-license.php</url>
         </license>
     </licenses>
+
+    <developers>
+        <developer>
+            <id>oleg_nenashev</id>
+            <name>Oleg Nenashev</name>
+            <email>o.v.nenashev@gmail.com</email>
+            <roles>
+                <role>maintainer</role>
+            </roles>
+        </developer>
+    </developers>
+
     <scm>
         <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-       <tag>HEAD</tag>
-   </scm>
-   <properties>
-       <jenkins.version>1.625.3</jenkins.version>
-       <java.level>7</java.level>
-   </properties>
-   <developers>
-       <developer>
-          <id>oleg_nenashev</id>
-          <name>Oleg Nenashev</name>
-          <email>o.v.nenashev@gmail.com</email>
-          <roles>
-              <role>maintainer</role>
-          </roles>
-       </developer>
-   </developers>
+        <tag>HEAD</tag>
+    </scm>
 
+    <properties>
```

Comment:
TBH I prefer keeping properties on the top so that they are easily accessible. when an update is needed